vassal-app/src/main/java/VASSAL/i18n/TranslateWindow.java

```diff
@@ -97,6 +97,9 @@ public class TranslateWindow extends JDialog implements ListSelectionListener,
   protected ConfigureTree myConfigureTree;
   protected CopyButton[] copyButtons;
 
+  protected JButton okButton;
+  protected JButton cancelButton;
```

Comment:
Could these be local?

Reply:
this one actually couldn't be local because it is needed in the initComponents method